Tensile/TensileCreateLibrary.py

```diff
@@ -1099,6 +1100,35 @@ def findLogicFiles(path: Path, logicArchs: Set[str], lazyLoading: bool, experime
 
     return list(str(l) for l in logicFiles)
 
+def createClientConfig(outputPath: Path, masterFile: Path, codeObjectFiles: List[str]) -> None:
+    """Outputs a client config file named best-solution.ini corresponding to a master library 
+       file and code-object parameters created by a TensileCreateLibrary invocation. Also
+       sets best-solution-mode to True.
+    
+    Args:
+        outputPath: The path to the tensile output directory where output files are written.
+        masterFile: Path to the master library file (.dat or .yaml).
+        codeObjectFiles: List of code object files created by TensileCreateLibrary.
+    """
+    iniFile = outputPath / "best-solution.ini"
+    
+    def param(key, value):
+      f.write("{}={}\n".format(key, value))
```

Comment:
Below you use `f"{...}"` f-string style formatting. Update here for consistency?